busted/modules/files/terra.lua

```diff
@@ -1,7 +1,10 @@
 local path = require 'pl.path'
 
 local ret = {}
-local ok, terralib = pcall(function() return require 'terralib' end)
+local ok, terralib = not not terralib, terralib --grab the injected global if it exists
+if not ok then
+  ok, terralib = pcall(require, 'terra') --otherwise, attempt to load terra as a shared library
+end
```

Comment:
I'd still rename `ok` to something like `terra_available` to make it more descriptive

Reply:
Should it really be silently ignoring it if terra isn't available? since Terra isn't on the default loaders list this code shouldn't even be invoked unless it is trying to load a terra test. Do you think it should be revised to send an error if terra isn't available and it is trying to load a terra test rather than the current behavior of silently dropping the test? Or should it be a warning?

Reply:
yes you're probably right, if terra was explicitly requested, but isn't available, it should error out.